README.md

```diff
@@ -46,7 +46,11 @@ To add an organization using AzureAD authentication, create a database organizat
 that includes an auth_type of `azureadv2` and an auth_config like the following ... 
 
 ```
-{}
+{
+    "TenantID": "edf3cc03-7edf-4299-871a-940bc318789c", 
+    "ClientSecret": "XO;a?UV.6rEC(d+sDhZ6:8*7%GqjVuw?", 
+    "ApplicationID": "4a01326d-5130-4326-92f6-fffe9f3dfc6d"
+}
```

Comment:
I guess the two lines below this should deleted or at least modified (i.e. no env vars for azure).

Reply:
I started to modify them, but wasn't sure what you were going to do with Microsoft social login. Would you like me to remove them until that's working?

Reply:
I think we will use a different auth type for theMicrosoft social login, which will have its own instructions.